internal/workflow/reconciler.go

```diff
@@ -126,7 +126,26 @@ func (r *Reconciler) processNewWorkflow(ctx context.Context, logger logr.Logger,
 
 // templateHardwareData defines the data exposed for a Hardware instance to a Template.
 type templateHardwareData struct {
-	Disks []string
+	Disks      []string
+	Interfaces []netInterface
```

Comment:
In the v1alpha2 API we're doubling down on clarity by using `NetworkInterfaces`. Should we do the same here?

Reply:
Sure thing.